### DISCUSSION TITLE
Welcome, NetJoxa!

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#119] [Subscribe]
Hola, @NetJox4!

After all the bits and struggles, I guess you finally deserve it! Congrats of your [x2 Galaxy Brain](https://github.com/NetJox4?achievement=galaxy-brain&tab=achievements) badge 😊🎉

**Comment by oscarxdev** [comment]
Thx, i just realized. The problem was that i had to uncheck my own answers.
